homeassistant/components/androidtv/config_flow.py

```diff
@@ -5,6 +5,7 @@
 import socket
 
 from androidtv import state_detection_rules_validator
+from androidtv.constants import CUSTOMIZABLE_COMMANDS
```

Comment:
The `CUSTOM_CURRENT_APP` entry in the `CUSTOMIZABLE_COMMANDS` set won't have any impact in this integration. You may want to define a constant that lacks this entry so that users aren't presented with an option that does nothing.

Reply:
It would make sense do this at library side, defining an additional constant like `HA_CUSTOMIZABLE_COMMANDS`? This allow to have potential new command available without the necessity to change the code.

Reply:
Ok, for now I implemented this here, just think about this.

Reply:
How about I add a sorted tuple `HA_CUSTOMIZABLE_COMMANDS`?

Reply:
Yes, it would be fine

Reply:
Thanks, I wait for your PR to be merged, than I will rebase and modify this.